### PR TITLE
create not update status field

### PR DESCRIPTION
**Pull request by huiwq1990**
Signed-off-by: huiwq1990 <huiwq1990@163.com>

### V. Checklist

- [x] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Comment by jasonliu747** [comment]
Hi @huiwq1990 thanks. Can you summarize this PR in description? Simply describe what this PR does briefly.

**Comment by huiwq1990** [comment]
> Hi @huiwq1990 thanks. Can you summarize this PR in description? Simply describe what this PR does briefly.

As `status` field is a sub resource, we should update it use `UpdateStatus` method. As we create the `nodemetric` cr by `create`, the assign will change anything.

**Review by jasonliu747 — COMMENTED** [review]
/lgtm
